nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -73,69 +83,70 @@ Below is a table documenting [corrected filter field changes](../release-notes/v
-| PowerFeed             | `region_id`         | instead of `/dcim/power-feeds/?region_id=<uuid>`, use `region=<uuid>`                  |
-|                       | `site_id`           | instead of `/dcim/power-feeds/?site_id=<uuid>`, use `site=<uuid>`                      |
-|                       | `power_panel_id`    | instead of `/dcim/power-feeds/?power_panel_id=<uuid>`, use `power_panel=<uuid>`        |
-|                       | `rack_id`           | instead of `/dcim/power-feeds/?rack_id=<uuid>`, use `rack=<uuid>`                      |
-| PowerPanel            | `region_id`         | instead of `/dcim/power-panels/?region_id=<uuid>`, use `region=<uuid>`                 |
-|                       | `site_id`           | instead of `/dcim/power-panels/?site_id=<uuid>`, use `site=<uuid>`                     |
-|                       | `rack_group_id`     | instead of `/dcim/power-panels/?rack_group_id=<uuid>`, use `rack_group=<uuid>`         |
-| PowerPort             | `region_id`         | instead of `/dcim/power-ports/?region_id=<uuid>`, use `region=<uuid>`                  |
-|                       | `device_id`         | instead of `/dcim/power-ports/?device_id=<uuid>`, use `device=<uuid>`                  |
-| Prefix                | `region_id`         | instead of `/ipam/prefixes/?region_id=<uuid>`, use `region=<uuid>`                     |
-|                       | `site_id`           | instead of `/ipam/prefixes/?site_id=<uuid>`, use `site=<uuid>`                         |
-| Site                  | `region_id`         | instead of `/dcim/sites/?region_id=<uuid>`, use `region=<uuid>`                        |
-| VirtualChassis        | `region_id`         | instead of `/dcim/virtual-chassis/?region_id=<uuid>`, use `region=<uuid>`              |
-|                       | `site_id`           | instead of `/dcim/virtual-chassis/?site_id=<uuid>`, use `site=<uuid>`                  |
-|                       | `master_id`         | instead of `/dcim/virtual-chassis/?master_id=<uuid>`, use `master=<uuid>`              |
-|                       | `tenant_id`         | instead of `/dcim/virtual-chassis/?tenant_id=<uuid>`, use `tenant=<uuid>`              |
-| VLANGroup             | `region_id`         | instead of `/ipam/vlan-groups/?region_id=<uuid>`, use `region=<uuid>`                  |
-|                       | `site_id`           | instead of `/ipam/vlan-groups/?site_id=<uuid>`, use `site=<uuid>`                      |
-| VLAN                  | `region_id`         | instead of `/ipam/vlans/?region_id=<uuid>`, use `region=<uuid>`                        |
-|                       | `site_id`           | instead of `/ipam/vlans/?site_id=<uuid>`, use `site=<uuid>`                            |
+|-----------------------|---------------------|------------------------------------------------------------------------------------------|
```

Comment:
I didnt really change this part, just added spaces at the end of the last column